### DISCUSSION TITLE
install-poetry vs get-poetry

### DISCUSSION THREAD

**Discussion by welrbraga** [python-poetry/poetry#4218] [Ideas]
Hi team,

I'm just a new trying to use poetry in my little projects and my first doubt was with these double installer scripts. One is suggested on official site and another here in git repo.

Both are available and I couldn't see any information about difference between them. But one of them is working fine the other no. Would please clarify the difference and when I have to use one or other, on docs, please!

These screenshots were taken on my Ubuntu 20.04 with Python 3.8.5 and the install-poetry.py, suggested here in Github repo, was not working, while get-poetry.py, suggest on <https://python-poetry.org/docs/> installed fine.

Regards,

![install-poetry-output](https://user-images.githubusercontent.com/7763513/123520088-621d1680-d685-11eb-99a4-a2d572e0748d.png)

![get-poetry-output](https://user-images.githubusercontent.com/7763513/123520090-65180700-d685-11eb-81f6-a80523143386.png)

**Comment by gh640** [comment]
`get-poetry.py` is old and `install-poetry.py` is quite new now. I guess currently most users still use `get-poetry.py` but will migrate to `install-poetry.py`. The following pages will help to know the situation:

- #3706
- https://python-poetry.org/history/#120a1---2021-05-21

**Reply by abn** [reply]
This is now available via https://install.python-poetry.org. 

https://github.com/python-poetry/install.python-poetry.org

You can install poetry via one of the following methods.

1. Install script (linked above).
2. Using [pipx](https://pypa.github.io/pipx/).
3. Distro provided packaging.
